tools/loader.md

```diff
@@ -60,7 +60,7 @@ Since tools like mysqldump will take us days to migrate massive amounts of data,
 
   -p string: the account and password of TiDB
 
-  -pprof-addr string: the pprof address of Loader. It tunes the performance of Loader (default: ":8272")
+  -status-addr string: the pprof address of Loader. It can also be used by Prometheus to pull Loader metrics
```

Comment:
update `the pprof address of Loader` too?
and `The pprof address of Loader` below